### PR TITLE
added ReleaseCargo command

### PR DESCRIPTION
**Pull request by Yoel-Perman-Brilliant**

### CODE REVIEW

File: src/main/java/frc/robot/Subsystems/Dispenser.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package frc.robot.Subsystems;
+
+import com.spikes2212.command.genericsubsystem.GenericSubsystem;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Dispenser extends GenericSubsystem {
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Subsystems/Dispenser.java:6]: extends `MotoredGenericSubsystem`

File: src/main/java/frc/robot/Subsystems/Transfer.java [hunk 2]
```diff
@@ -1,4 +1,11 @@
 package frc.robot.Subsystems;
 
-public class Transfer {
+import com.spikes2212.command.genericsubsystem.GenericSubsystem;
+
+public class Transfer extends GenericSubsystem {
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Subsystems/Transfer.java:5]: ditto

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Commands/ReleaseCargo.java [hunk 3]
```diff
@@ -1,4 +1,27 @@
...
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Subsystems.Dispenser;
+import frc.robot.Subsystems.Transfer;
+
+
+/**
+ * @author Yoel Perman Brilliant
+ */
+public class ReleaseCargo extends SequentialCommandGroup {
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Commands/ReleaseCargo.java:13]: add documentations

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Commands/ReleaseCargo.java [hunk 4]
```diff
@@ -1,4 +1,29 @@
...
+ * A command which releases the cargo into the lower hub using the {@link Transfer} and {@link  Dispenser} subsystems.
+ * @author Yoel Perman Brilliant
+ */
+public class ReleaseCargo extends SequentialCommandGroup {
+
+    public ReleaseCargo() {
+        Dispenser dispenser = Dispenser.getInstance();
+        Transfer transfer = Transfer.getInstance();
+        addCommands(
+                new MoveGenericSubsystem(dispenser, Dispenser.SPEED).withTimeout(Dispenser.DISPENSER_WARM_UP_TIME),
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Commands/ReleaseCargo.java:20]: Use the Dispenser's Encoder with `MoveGenericSubsystemPIDForSpeed`

File: src/main/java/frc/robot/Commands/ReleaseCargo.java [hunk 5]
```diff
@@ -1,4 +1,29 @@
...
+                new ParallelCommandGroup(
+                        new MoveGenericSubsystem(dispenser, Dispenser.SPEED),
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Commands/ReleaseCargo.java:22]: ditto

File: src/main/java/frc/robot/Subsystems/Dispenser.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package frc.robot.Subsystems;
+
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+
+public class Dispenser extends MotoredGenericSubsystem {
+    public static final double SPEED;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Subsystems/Dispenser.java:6]: remove

File: src/main/java/frc/robot/Subsystems/Dispenser.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+    public static final double DISPENSER_WARM_UP_TIME = 3;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Subsystems/Dispenser.java:7]: remove

File: src/main/java/frc/robot/Subsystems/Dispenser.java [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+    public static final double TRANSFER_PLUS_DISPENSE_TIME = 5;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Subsystems/Dispenser.java:8]: remove

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Comment by Yoel-Perman-Brilliant** [comment]
haven't deleted the branch in case we need it in the future

**Pull request closed without merging**